ngraph/test/onnx/onnx_test_utils.in.cpp

```diff
@@ -25,6 +27,7 @@ using namespace ngraph;
 static std::string s_manifest = "${MANIFEST}";
 using TestEngine = test::ENGINE_CLASS_NAME(${BACKEND_NAME});
 
+#ifdef NGRAPH_ONNX_EDITOR_ENABLE
```

Comment:
Let's move `ONNXModelEditor` tests to a separate file.

Reply:
These are no `ONNXModelEditor` tests, but `TestCase` class tests that use the editor to prepare test data (we test different input types combinations).
I configured cmake to build `onnx_test_utils.in.cpp` only if `NGRAPH_ONNX_EDITOR_ENABLE=ON`